war/src/main/less/modules/table.less

```diff
@@ -189,8 +190,8 @@
       svg,
       .build-status-icon__wrapper,
       img {
-        width: 16px !important;
-        height: 16px !important;
+        height: 1rem !important;
+        max-width: 1.2rem !important;
```

Comment:
the build status icons are mis-aligned on the Medium table size:

![image](https://user-images.githubusercontent.com/21194782/193589658-202146eb-a8cf-4035-8b6c-99818f5dc33d.png)